pages/advanced/api-safe-transaction-service/guides/api.md

```diff
@@ -0,0 +1,627 @@
+# Interacting with API
+
+This section shows some examples of how to interact with the Safe Transaction Service API in different situations.
+
+The examples have been proposed using [Curl](https://github.com/curl/curl) requests, the [Safe Apps SDK](https://github.com/safe-global/safe-apps-sdk) and the [safe-eth-py](https://github.com/safe-global/safe-eth-py) python library.
+
+## Prerequisites
+
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-nodejs-and-npm).
+2. [Python](https://www.python.org/downloads/) >= 3.9 (Python 3.12 is recommended).
+3. Have a Safe configured with threshold equal to 2, where 2 signatures are needed to execute transactions.
+
+## Install dependencies
+
+```bash
```

Comment:
@germartinez @tanay1337 Could it be posible to have tabs like other docs, one tab for `curl`, other `Python` and another `Typescript`?